WalletWasabi/WabiSabi/Client/CoinJoin/Manager/CoinJoinManager.cs

```diff
@@ -160,12 +168,25 @@ private async Task MonitorAndHandleCoinJoinsAsync(CancellationToken stoppingToke
 
 	private async Task HandleCoinJoinCommandsAsync(ConcurrentDictionary<WalletId, CoinJoinTracker> trackedCoinJoins, ConcurrentDictionary<IWallet, TrackedAutoStart> trackedAutoStarts, CancellationToken stoppingToken)
 	{
-		var coinJoinTrackerFactory = new CoinJoinTrackerFactory(HttpClientFactory, RoundStatusUpdater, CoinJoinConfiguration, stoppingToken);
+		CoinJoinTrackerFactory? coinJoinTrackerFactory = null;
+		if (!HasCoordinatorConfigured)
+		{
+			coinJoinTrackerFactory = new CoinJoinTrackerFactory(HttpClientFactory!, RoundStatusUpdater!, CoinJoinConfiguration, stoppingToken);
+		}
```

Comment:
This makes no sense. Please take a look at:
https://github.com/WalletWasabi/WalletWasabi/commit/dc71b6fd9736922fd5bc31123f57c6384fb17682